scripts/deploy.js

```diff
@@ -80,7 +80,7 @@ async function run() {
       console.error(`Run with --restart to restart the bot.`);
     }
 
-    await ssh.exec(`docker pull brigand/js-eval`);
+    await ssh.exec(`node src/plugins/js-eval/init`); // build brigand/js-eval image
```

Comment:
How long does this take to execute if the image is unchanged?

Reply:
That's about 7 seconds longer than it needs to be :-p Could we try this?

- hash the docker file
- compare to Dockerfile.hash on the server (if it exists)
- if not equal, build the container and if successful write the hash to Dockerfile.hash

Reply:
hmm, a bit hard to do (above all since I've added `run.js` COPY'ed from `Dockerfile`). Does it really matter? the deploy don't need to be frequent

Note that if you can provide a `GH_TOKEN` env var during the build, it fetches the latest nodejs version automatically

Reply:
Yeah, that's fine. Also the cron is a good idea.